charts/argo-events/templates/sensor-controller-deployment.yaml

```diff
@@ -72,3 +72,7 @@ spec:
       {{- if .Values.sensorController.affinity }}
       affinity: {{ toYaml .Values.sensorController.affinity | nindent 8 }}
       {{- end }}
+      {{- if .Values.sensorController.resources }}
+      resources:
```

Comment:
`resources` is not a pod-level setting. Please move this inside the `containers` section.